clients/ts/signalr/tests/HttpConnection.test.ts

```diff
@@ -337,31 +305,30 @@ describe("HttpConnection", () => {
                     return "";
                 }
                 return new HttpResponse(204, "No Content", "");
-            });
+            })
+            .on("DELETE", (r) => new HttpResponse(202));
```

Comment:
Was the test http client's lack of handling deletes not showing up in the old test framework?

Reply:
No, I added `try { ... } finally { await connection.stop(); }` to clean up background things during tests and it meant `DELETE` requests started being issued.